src/ui/public/agg_types/controls/latest.html

```diff
@@ -0,0 +1,29 @@
+<div class="form-group">
+  <label for="sort">
+    Sort On
+  </label>
+
+  <select
+    class="form-control"
+    name="sort"
+    required
+    ng-model="agg.params.sort"
+    ng-show="indexedFields.length"
+    auto-select-if-only-one="indexedFields"
+    ng-options="field as field.displayName group by field.type for field in indexedFields">
+  </select>
+</div>
+
+<div class="form-group">
+  <label>
+    Order
+  </label>
+
+  <select
+     name="order"
+     ng-model="agg.params.order"
```

Comment:
Hmm, this is a bit nitpicky of me, but I think the sort, and sort order should use the same property names, since they are dependent on one another, and I would hate for someone later to not realize that.

For instance you could call the sort field `agg.params.sort.field` and the order `agg.params.sort.dir` or something to that effect. but with that it's clear they're both dealing with the sort, no matter where it's put.